### PR TITLE
Ticket #19249

### PR DESCRIPTION
**Pull request by hans-vm**
Added easy replicate option to volume/dataset

### CODE REVIEW

File: ui/sections/storage/inspectors/volume-dataset.reel/volume-dataset.js [hunk 1]
```diff
@@ -77,12 +77,15 @@ exports.VolumeDataset = AbstractInspector.specialize(/** @lends VolumeDataset# *
...
             this.snapshotType = this._sectionService.VOLUME_SNAPSHOT_TYPE;
-            this._sectionService.listVolumeSnapshots().then(function(snapshots) {
-                self.snapshots = snapshots;
-            });
             this.vmwareDatasetType = this._sectionService.VMWARE_DATASET_TYPE;
-            this._sectionService.listVmwareDatasets().then(function(vmwareDatasets) {
+            return Promise.all([
+                this._sectionService.listVolumeSnapshots(),
+                this._sectionService.listVmwareDatasets(),
+                this.application.dataService.getNewInstanceForType(Model.ReplicationOptions)
```
Comment (thibaultzanini) [inline ui/sections/storage/inspectors/volume-dataset.reel/volume-dataset.js]: We try to go away from that design, the storage-section-service should have a method that return a new instance of ReplicationOptions. :)

**Review by thibaultzanini — CHANGES_REQUESTED** [review]

**Review by thibaultzanini — APPROVED** [review]
LGTM

**Comment by thibaultzanini** [comment]
@pchaussalet Can you double check this one please.

**Comment by pchaussalet** [comment]
Looks good to me

**Pull request merged**